Makefile

```diff
@@ -169,6 +169,10 @@ target/bundle/config/authority.json.template.example:
 
 target/bundle/config: target/bundle/config/lrsql.json.example target/bundle/config/authority.json.template.example
 
+target/bundle/customization:
```

Comment:
in order to work this would need to be _at least_ `target/bundle/lrsql/customization` to maintain the relative resource path. As I mentioned on slack I'd also favor an additional wrapping directory so just that can be included in classpath overrides.